test/clj/puppetlabs/trapperkeeper/services/webserver/jetty9_service_handlers_test.clj

```diff
@@ -191,11 +191,18 @@
             server-binary-messages (atom [])
             client-messages        (atom [])
             client-binary-messages (atom [])
+            client-request-path    (atom "")
+            client-remote-addr     (atom "")
+            client-is-ssl          (atom false)
```

Comment:
Maybe it would be good to have the default be something that the `ssl?` function wouldn't return, like `nil`?  Then if the `on-connect` handler did not get invoked, the assertion comparing `client-is-ssl` to `false` would blow up?  Suppose some of the other assertions would blow up anyway in that case, so probably not a big deal.